app/assets/javascripts/application/judgement.js

```diff
@@ -0,0 +1,35 @@
+var judgement = (function(Rx, $) {
```

Comment:
Using either 2 or 4 spaces is fine ([Style guide](https://github.com/airbnb/javascript#whitespace) suggests using 2), but try not to use both in the same file.

Reply:
True, I didn't notice the indentation was four spaces in this file. I've changed to two spaces everywhere to be a bit more consistent.